<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>huffman</class>
 <widget class="QDialog" name="huffman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="CompressButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сжать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutComEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OutputComLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите полное имя файла вывода</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InComEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InputComLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите полное имя файла ввода</string>
   </property>
  </widget>
  <widget class="QLabel" name="CompressLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сжатие</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="DecompressButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Разжать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutDecomEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OutDecomLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите полное имя файла вывода</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InDecomEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InputDecomLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите полное имя файла ввода</string>
   </property>
  </widget>
  <widget class="QLabel" name="DecompressLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Разжатие</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
